packages/block-library/src/query/block.json

```diff
@@ -34,13 +34,15 @@
 	"providesContext": {
 		"queryId": "queryId",
 		"query": "query",
-		"layout": "layout"
+		"layout": "layout",
+		"align": "align"
 	},
 	"usesContext": [
 		"postId"
 	],
 	"supports": {
-		"html": false
+		"html": false,
+		"align": [ "wide", "full" ]
```

Comment:
I don't really understand why the "query" block supports these flags? This is not a block that has any markup?

Reply:
Check my comment about the attributes. This is what we had discussed about `layout` as well, so `align` goes to `Query` now.

Reply:
If we want to avoid all "layout" props from being present in the query we need something in the vein of #7694 for the outer wrapper